suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml

```diff
@@ -0,0 +1,122 @@
+---
+# ==================================================================================
+# Tier-level: 2
+# Test-Suite: tier-2_cephadm_conf_rgw_with_warn_state.yaml
+# Cluster Configuration:
+#    conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml
+# Test Scenarios:
+#   Bootstrap cluster using cephadm
+#   Configure RGW in warn state
+#   Add OSD nodes
+#   Perform object operations
+# ==================================================================================
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                registry-url: registry.redhat.io
+                mon-ip: node1
+                initial-dashboard-password: admin@123
+                dashboard-password-noupdate: true
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          # Configure RGW at cluster warn state (OSD count 0 < osd_pool_default_size 3)
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: bootstrap and configure RGW in warn state
+      destroy-cluster: false
+      polarion-id: CEPH-83573835
+      module: test_cephadm.py
+      name: Deploy cluster using cephadm and configure RGW in warn state.
+  - test:
```

Comment:
Add comments for each test case. Include purpose of TC.

Reply:
Thanks @vamahaja 
Updated